cime_config/machines/cmake_macros/crayclang-scream_frontier-scream-gpu.cmake

```diff
@@ -1,6 +1,6 @@
 set(MPICC "mpicc")
-set(MPICXX "hipcc")
-set(MPIFC "ftn")
+set(MPICXX "mpicxx") # Needs MPICH_CXX to use hipcc
+set(MPIFC "ftn") # Linker needs to be plain ftn, not mpif90
```

Comment:
Here I would remove the "plain ftn" part because `ftn` is a Cray wrapper, not the bare Fortran compiler.

Reply:
@ambrad , good call. I have improved the text of the comment.